dnf-behave-tests/dnf/security-upgrade.feature

```diff
@@ -13,6 +13,36 @@ Given I use repository "security-upgrade"
       | upgrade       | json-c-0:2-2.x86_64         |
 
 
+@bz2124483
+# This scenario is the same as the one above just with noarch packages present in the
+# repo and advisory instead of i686 this causes different behavior - it fails.
+# Technically we could get bz2088149 again with just a different package set.
+# However while this is possible it shouldn't happen because noarch packages are special - they work
+# on all architectures and we shouldn't see noarch builds together with arch specific builds as it
+# is setup in this scenario (in repo security-upgrade-noarch).
+Scenario: --security upgrade with advisories with pkgs of different arches (noarch variant)
+Given I use repository "security-upgrade-noarch"
+  And I successfully execute dnf with args "install json-c-1-1 bind-libs-lite-1-1"
```

Comment:
Please could you add then step with check of the transaction table? It will be more transparent what (architecture) is installed on the system

Reply:
Sure, added.
I have also added a link to a PR with more details about the tests situation.

Reply:
Perfect